File: src/main/java/com/pebble/baseAuth/domain/UserRole.java
package com.pebble.baseAuth.domain;

import lombok.Getter;
import lombok.RequiredArgsConstructor;

@Getter
@RequiredArgsConstructor
public enum UserRole {
    // 일반 사용자 권한
    ROLE_USER("일반 사용자"),
    
    // 관리자 권한
    ROLE_ADMIN("시스템 관리자");

    private final String description;
}

File: auth-module/src/main/kotlin/com/pebble/basicAuth/domain/UserRole.kt
package com.pebble.basicAuth.domain

enum class UserRole(val description: String) {
    // 일반 사용자 권한
    ROLE_USER("일반 사용자"),

    // 관리자 권한
    ROLE_ADMIN("시스템 관리자");
}

